checks/config.go

```diff
@@ -57,12 +59,24 @@ type Config struct {
 	// []string{".*", "_*"}.  This is a glob that is applied to each path
 	// component of each file.
 	IgnorePatterns []string `yaml:"ignore_patterns"`
+
+	// MaxConcurrent, if not zero, is the maximum number of concurrent processes
```

Comment:
NP, bits are still in-tact :) ATM this is using `yaml:"-"`, so it isn't loaded from the config file. The main command sets this. I'll check in a rebase soon.